.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 wasm-pack.log
+/.vscode
```

Comment:
I don't mind having this here, but you probably want to put this in your home directory's `.gitignore` so that you don't have to add it to every project you contribute to ;)

Reply:
Some projects I work on might want to have a .vscode config which is why I don't have it in my home directory's .gitignore file. If you're fine with it here, it might be nice to keep.